MapboxDirections/MBRouteOptions.swift

```diff
@@ -315,6 +316,8 @@ open class RouteOptions: NSObject, NSSecureCoding {
      */
     open var locale: Locale?
     
+    private let supportedLocales: [String] = ["de", "en", "es", "fr", "id", "nl", "ru", "sv", "vi", "zh-hans"]
```

Comment:
If the locale is Simplified Chinese, its locale identifier will be `zh-Hans`, not `zh-hans`. The Directions API accepts `zh-Hans` as a `language`.